app/src/main/java/io/left/meshim/database/Migrations.java

```diff
@@ -60,7 +60,16 @@ public void migrate(SupportSQLiteDatabase database) {
             database.execSQL("DROP TABLE OldMessages");
         }
     };
+    public static final Migration MIGRATION_6_7 = new Migration(6, 7) {
+        @Override
+        public void migrate(SupportSQLiteDatabase database) {
+            database.execSQL("ALTER TABLE Messages "
+                    + " ADD COLUMN FilePath TEXT DEFAULT ''");
+            database.execSQL("ALTER TABLE Messages "
+                    + " ADD COLUMN FileExtension TEXT DEFAULT ''");
```

Comment:
I expect MIME type would be more useful to save than extension. Extensions can vary a lot (e.g. `JPEG` vs `jpg`) and are ultimately useless compared to the encoding.